package.json

```diff
@@ -34,6 +34,7 @@
     "eth-phishing-detect": "^1.2.0",
     "functional-red-black-tree": "^1.0.1",
     "i18n-js": "^3.3.0",
+    "jsc-android": "250230.2.1",
```

Comment:
Hmm, I'm curious @hysekbutt , why was this included? Can you elaborate a bit please?

Reply:
reverted